content/en/metrics/_index.md

```diff
@@ -111,6 +128,34 @@ Space aggregation splits a single metric into multiple timeseries by tags such a
 
 There are four aggregators that can be applied when using space aggregation: _sum_, _min_, _max_, and _avg_. Using the above example, say that your hosts are spread across four regions: us-east-1, us-east-2, us-west-1, and us-west-2. The hosts in each region need to be combined using an aggregator function. Using the _max_ aggregator would result in the maximum latency experienced across hosts in each region, while the _avg_ aggregator would yield the average latency per region.
 
+### Combining time series
+
+Time series are often combined together to produce a single representative time series. For example, you might want to see the average data received by the web servers in your infrastructure.
+
+Take two hosts submitting the same metric to Datadog:
+
+{{< img src="metrics/introduction/adding-by-host.png" alt="Two hosts send metrics to Datadog"  style="width:35%;">}}
+
+When you look at the data separated by host, `net.bytes_rcvd` is submitted at slightly different times:
+
+{{< img src="metrics/introduction/mismatched-time-series.png" alt="Mismatched Time Series" >}}
+
```

Comment:
This seems like it might be useful information, but I'm not convinced it belongs on this page. Maybe it could be explained and expanded into a guide or step by step task and placed elsewhere. 

```suggestion

```